packages/shared_preferences/shared_preferences_foundation/darwin/Classes/SharedPreferencesPlugin.swift

```diff
@@ -10,6 +10,8 @@ import Flutter
 import FlutterMacOS
 #endif
 
+let _defaultPrefix: String = "flutter."
```

Comment:
No underscore (https://google.github.io/swift/#naming-conventions-are-not-access-control)

Also, this should probably be a `static let` inside the class, with `getAllPrefs` moved inside the class as well.